rpm/btrfs-balancer.spec

```diff
@@ -1,14 +1,18 @@
 Name:		btrfs-balancer
 Summary:	Automatic balancing service for btrfs filesystem
-Version:	1.1.0
+Version:	1.2.0
 Release:	1
 Group:		System/Filesystems
 License:	BSD
-URL:		https://bitbucket.org/jolla/tools-atruncate
+URL:		https://github.com/sailfishos/btrfs-balancer
 Source0:	%{name}-%{version}.tar.bz2
 
+BuildRequires:  pkgconfig(Qt5Core)
+BuildRequires:  pkgconfig(Qt5DBus)
+BuildRequires:  pkgconfig(keepalive)
+Requires:       sailfish-snapshot
```

Comment:
Not ok to depend on sailfish-snapshot - at least not for now. Release schedules don't match.